input/templates/my-account-address-book.json

```diff
@@ -32,8 +32,7 @@
 			"edit" : "Edit",
 			"delete" : "Delete"
 		},
-		"additionalAddressTitle" : "Additional Address",
-		"additionalAddress" : {
+		"secondaryAddress" : {
```

Comment:
As far as I know, this should be a list of address. So to make it clear, the page has on the left the "Default Shipping Address", below the "Default Billing Address", and on the right there is a list of additional addresses.